crt_portal/static/css/internet-explorer.css

```diff
@@ -0,0 +1,2 @@
+[hidden]{display:none!important}
```

Comment:
Shim from here: https://cloudfour.com/thinks/see-no-evil-hidden-content-and-accessibility/